src/main/java/hudson/plugins/jira/versionparameter/JiraVersionParameterDefinition.java

```diff
@@ -159,5 +185,12 @@ public Result(final Version version) {
             this.name = version.getName();
             this.id = version.getId();
         }
+
+        @Override
+        public boolean equals(Object o) {
+            if (o == null || getClass() != o.getClass()) return false;
+            Result result = (Result) o;
+            return Objects.equals(name, result.name) && Objects.equals(id, result.id);
```

Comment:
Adjust tests, this is important method that needs to be tested (if needed at all)

Reply:
I am not sure why is it still saying that 189 is not covered. 
When I run the test with coverage in intellij, it shows this method is fully covered.

Reply:
I think the missing branch might be comparing comparing objects with the same name but different IDs (or the other way round) - I don't see this test case.